src/devtools/useAtomsDevtools.ts

```diff
@@ -52,11 +52,16 @@ const getDevtoolsState = (atomsSnapshot: AtomsSnapshot) => {
   }
 }
 
-export function useAtomsDevtools(name: string, scope?: Scope) {
-  const ScopeContext = getScopeContext(scope)
+interface DevtoolsOptions {
+  scope?: Scope
+  enabled?: boolean
+}
+
+export function useAtomsDevtools(name: string, options?: DevtoolsOptions) {
```

Comment:
Please make it backward compatible.

```ts
export function useAtomsDevtools(name: string, options?: DevtoolsOptions)
/**
 * @deprecated Please use object options
 */
export function useAtomsDevtools(name: string, options?: Scope)
export function useAtomsDevtools(name: string, options?: DevtoolsOptions | Scope) {
  if (typeof options !== 'undefined' && typeof options !== 'object') {
    console.warn('DEPRECATED...')
    options = { scope: options }
```

Check https://github.com/pmndrs/valtio/pull/400 for reference.